src/commands/remove.rs

```diff
@@ -49,6 +50,10 @@ pub async fn remove(
         handler.queue().modify_queue(|v| {
             v.remove(remove_index);
         });
+
+        drop(handler);
+        update_queue_messages(&ctx.http, &ctx.data, &call, guild_id).await;
```

Comment:
Again, maybe first send the feedback to the user ("Removed!") and then update the queue embed